File: app/src/main/java/br/com/patrocine/patrocine/model/Grid.java
package br.com.patrocine.patrocine.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Grid implements Serializable {

    @SerializedName("id")
    private int id;
    @SerializedName("movie_id")
    private int MovieId;
    @SerializedName("date")
    private String date;
    @SerializedName("room")
    private String room;
    @SerializedName("category")
    private String category;
    @SerializedName("language")
    private String language;
    @SerializedName("day")
    private String day;
    @SerializedName("hour")
    private String hour;

    public Grid(){
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getMovieId() {
        return MovieId;
    }

    public void setMovieId(int movieId) {
        MovieId = movieId;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getRoom() {
        return room;
    }

    public void setRoom(String room) {
        this.room = room;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getLanguage() {
        return language;
    }

    public void setLanguage(String language) {
        this.language = language;
    }

    public String getDay() {
        return day;
    }

    public void setDay(String day) {
        this.day = day;
    }

    public String getHour() {
        return hour;
    }

    public void setHour(String hour) {
        this.hour = hour;
    }
}

File: app/src/main/java/br/com/patrocine/patrocine/model/Grid.kt
package br.com.patrocine.patrocine.model

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.SerializedName

class Grid() : Parcelable {

    @SerializedName("id")
    var id: Int = 0
    @SerializedName("movie_id")
    var MovieId: Int = 0
    @SerializedName("date")
    var date: String? = null
    @SerializedName("room")
    var room: String? = null
    @SerializedName("category")
    var category: String? = null
    @SerializedName("language")
    var language: String? = null
    @SerializedName("day")
    var day: String? = null
    @SerializedName("hour")
    var hour: String? = null



    constructor(parcel: Parcel) : this() {
        id = parcel.readInt()
        MovieId = parcel.readInt()
        date = parcel.readString()
        room = parcel.readString()
        category = parcel.readString()
        language = parcel.readString()
        day = parcel.readString()
        hour = parcel.readString()
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeInt(id)
        parcel.writeInt(MovieId)
        parcel.writeString(date)
        parcel.writeString(room)
        parcel.writeString(category)
        parcel.writeString(language)
        parcel.writeString(day)
        parcel.writeString(hour)
    }

    override fun describeContents(): Int {
        return 0
    }

    override fun toString(): String {
        return "Grid(id=$id, MovieId=$MovieId, date=$date, room=$room, category=$category, language=$language, day=$day, hour=$hour)"
    }

    companion object CREATOR : Parcelable.Creator<Grid> {
        override fun createFromParcel(parcel: Parcel): Grid {
            return Grid(parcel)
        }

        override fun newArray(size: Int): Array<Grid?> {
            return arrayOfNulls(size)
        }
    }

}